onnxruntime/core/common/str_helper.cc

```diff
@@ -32,5 +32,23 @@ std::wstring ToWideString(const std::string& s) {
   return ret;
 }
 
+std::vector<std::string> SplitStr(const std::string& s, const char separator) {
```

Comment:
Just curious,
If it's a common function, would it be performance aware?

Would `find+string_view` be good?

Reply:
No, it is not - the usage is mostly about parsing configure strings on session start.
Curious too - why would find + string_view be faster? ;-)

Reply:
It would be faster because string_view does not allocate memory and does not copy data.